### ISSUE TITLE
Dark mode color's scheme is currently unreadable - Feature request

### ISSUE THREAD

**Issue by alexandre-mahdhaoui** [callumlocke/json-formatter#192]
Since last release of google chrome, it seems that the browser is trying to apply dark mode to Json response. 
Is it possible to detect the dark mode & at least put a white color font to it ?

**Comment by FilipChalupa** [comment]
This is how it looks now in dark theme:

![dark](https://user-images.githubusercontent.com/1045362/166158894-12367675-ecfc-4c64-a295-cd8b61c4baed.png)

And light:

![light](https://user-images.githubusercontent.com/1045362/166158911-fc9e02af-7654-4b67-a3c8-e777dda7792f.png)

**Comment by punkosu** [comment]
Alternatively is there any way to disable dark mode for these raw JSON views? That would be my preference. White text would be ok as well, but I'd prefer not having dark mode at all.

**Comment by vlaube** [comment]
up

**Comment by fr-rose-steven** [comment]
I tried to fix it temporarily myself by just adding "background:#fff;" to the body in the content.css
I loaded the extension as described but although the css is seemingly added from the extension, it's missing my background statement.
I have no idea why. If I add it via dev console manually after the page loaded, it works.
No idea where the css comes from.
Any suggestions?

**Comment by vaibhavrathod** [comment]
Please help, Is there any way to disable dark mode for raw JSON views?

**Comment by fr-rose-steven** [comment]
@vaibhavrathod this PR worked for me: https://github.com/callumlocke/json-formatter/pull/193

**Comment by waterfall531** [comment]
up

**Comment by zhangjing1997** [comment]
> @vaibhavrathod this PR worked for me: #193

also worked for me, tried OK by following Installation#Option2 on the readme.

**Comment by marjorg** [comment]
Based on the lack of activity in this project, I'm assuming this will never be fixed. Been trying out this one that works great in dark mode https://jsonview.com/

**Comment by PythonicNinja** [comment]
same issue on my side

**Comment by PythonicNinja** [comment]
Alternative that seems to be working fine.
https://chrome.google.com/webstore/detail/json-viewer/gbmdgpbipfallnflgajpaliibnhdgobh?hl=en

Has multiple theme support and works fine with big jsons.

**Comment by juanjoms** [comment]
A temporary hack taken from #193:
Open dev tools and paste following line:
`document.querySelector('meta[name="color-scheme"]').setAttribute("content", "light");`
it will change to light theme

**Comment by lostllama** [comment]
On the one hand I have the same issue, but on the other: this repository hasn't been updated in 7+ years. I don't expect this will get fixed.

**Comment by ReyArlena** [comment]
I agree, doubtful it's going to be fixed 

My workaround in case it's of interest to anyone - if you do wish to keep using this extention, and use dark mode, you can use something like [Stylus](https://chrome.google.com/webstore/detail/stylus/clngdbkpkpeebahjckkjfobafhncgmne) to add your own CSS.

- Click the extension > Manage
- Under Actions > Write New Style
- Under "Enter a Name/URL" just put `*` for wildcard / all domains
- Paste in the below CSS or make your own, and click Save

```css
.k {
    color: white;
}

.s {
    color: #27d714;
}

.bl, .nl, .n {
    color: #3292e5;
}
```

Depending on how many sites you browse with similar classes, you may not want to use a wildcard for the domain, etc

Also there are other similar browser extentions, just check you're happy with the permissions they require before installing them 🙂

**Comment by kriskate** [comment]
thanks for the easy fix @ReyArlena , I've added this to [Stylish](https://userstyles.org/) and I can read my jsons again!

**Comment by LMS007** [comment]
dang, this needs to get fixed :(

**Comment by viniciussanchez** [comment]
Up

**Comment by MatheusR42** [comment]
up!

**Comment by dangkaka** [comment]
same issue

**Comment by luukdv** [comment]
I've tried a few alternatives since this repo seems unmaintained indeed, and settled on this one: [JSON Viewer Pro - A completely free extension to visualise JSON response in awesome Tree and Chart view with great user experience and options](https://chrome.google.com/webstore/detail/json-viewer-pro/eifflpmocdbdmepbjaopkkhbfmdgijcc)

**Comment by danlavrushko** [comment]
a fix was already merged in #191 and submitted to Chrome Web Store for review 🎉 
https://github.com/callumlocke/json-formatter/pull/191#issuecomment-1125461589
